### PR TITLE
Page weight queries

### PR DESCRIPTION
**Pull request by khempenius**
Closes https://github.com/HTTPArchive/almanac.httparchive.org/issues/99

### CODE REVIEW

File: sql/2019/18_Page_Weight/18_01.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# 18_01: Distribution of page weight by resource type and client.
```
[the remark this answers is not in the document]
Comment (rviscomi) [reply sql/2019/18_Page_Weight/18_01.sql]: Ping

**Comment by khempenius** [comment]
@rviscomi finally added :)

File: sql/2019/18_Page_Weight/18_01c.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+SELECT
+  ROUND(AVG(bytesTotal) / 1024) AS total_kbytes,
+  ROUND(AVG(bytesHtml) / 1024) AS html_kbytes,
+  ROUND(AVG(bytesJS) / 1024) AS js_kbytes,
+  ROUND(AVG(bytesCSS) / 1024) AS css_kbytes,
+  ROUND(AVG(bytesImg) / 1024) AS img_kbytes,
+  ROUND(AVG(bytesOther) / 1024) AS other_kbytes,
+  ROUND(AVG(bytesHtmlDoc) / 1024) AS html_doc_kbytes
+FROM
+  `httparchive.summary_pages.2019_07_01_*`
```
Comment (rviscomi) [inline sql/2019/18_Page_Weight/18_01c.sql]: Also group by client so we can get desktop vs mobile.

File: sql/2019/18_Page_Weight/18_01c.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#standardSQL
+# 18_01c: Average size of each of the resource types.
+SELECT
+  ROUND(AVG(bytesTotal) / 1024) AS total_kbytes,
```
Comment (rviscomi) [inline sql/2019/18_Page_Weight/18_01c.sql]: Round to 2 decimal places, for each of these
```suggestion
  ROUND(AVG(bytesTotal) / 1024, 2) AS total_kbytes,
```

File: sql/2019/18_Page_Weight/18_03.sql [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#standardSQL
+# 18_03: Distribution of response size by response format
+SELECT
+  percentile,
```
Comment (rviscomi) [inline sql/2019/18_Page_Weight/18_03.sql]: Just noticed this one should also be selecting `_TABLE_SUFFIX AS client` and `GROUP BY client`

File: sql/2019/18_Page_Weight/18_04.sql [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#standardSQL
+# 18_04: Distribution of response size by response type
+SELECT
```
Comment (rviscomi) [inline sql/2019/18_Page_Weight/18_04.sql]: Ditto on `_TABLE_SUFFIX`

**Comment by khempenius** [comment]
Updated.

**Pull request merged**